src/Hydrators/IngredientEntityHydrator.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+class IngredientEntityHydrator {
+    private $db;
+    public function __construct(PDO $db) {
```

Comment:
Would the typehinting here not be 'Db' instead of 'PDO'? We're creating our own Database class, so it makes sense to use that.